README.md

```diff
@@ -69,7 +69,6 @@ cd ops
 export COMPOSE_DOCKER_CLI_BUILD=1
 export DOCKER_BUILDKIT=1
 docker-compose build
-docker-compose up -d
```

Comment:
You'll also want to remove the "bring the stack up and wait for sequencer" part in the github workflow.